pkg/cmd/profile/use.go

```diff
@@ -12,50 +12,92 @@ import (
 
 	"github.com/daytonaio/daytona/pkg/cmd/output"
 	"github.com/daytonaio/daytona/pkg/views"
+	"github.com/daytonaio/daytona/pkg/views/profile"
 	"github.com/spf13/cobra"
 )
 
 var ProfileUseCmd = &cobra.Command{
 	Use:   "use",
 	Short: "Set the active profile",
```

Comment:
Please change "Set the active profile" to "Use profile [PROFILE_NAME]" - the description is more important now that the help is displayed more often, so we can fix it as a part of this PR

Reply:
Updated my PR, making change in docs is very easy. Thanks for generate-cli-docs.sh